taxonomies/nutrients.txt

```diff
@@ -5653,7 +5659,7 @@ pa:ਸਿਲੀਕਾਨ ਡਾਈਆਕਸਾਈਡ
 pl:ditlenek krzemu
 # pnb:سلیکان ڈائیآکسائیڈ
 ps:سليكان ډای اکسايډ
-pt:dióxido de silício
+pt:Sílica, Dióxido de silício, SiO2 
```

Comment:
Same as before, is there any poduct specifying "SiO2" as a nutrient?

Reply:
Please see the previous image.